src/ray/object_manager/object_manager.h

```diff
@@ -381,13 +388,19 @@ class ObjectManager : public ObjectManagerInterface {
   /// all incoming object transfers.
   std::vector<std::thread> receive_threads_;
 
+  /// Reference to Plasma Store.
+  plasma::PlasmaClient &store_client_;
+
   /// Connection pool for reusing outgoing connections to remote object managers.
   ConnectionPool connection_pool_;
 
   /// Mapping from locally available objects to information about those objects
   /// including when the object was last pushed to other object managers.
   std::unordered_map<ObjectID, LocalObjectInfo> local_objects_;
 
+  /// Set of objects created from inlined data.
```

Comment:
```suggestion
  /// Set of objects created from inlined data retrieved from the GCS. Objects in this map will not be added to the GCS object table since a GCS entry with the inlined data already exists.
```